.travis.yml

```diff
@@ -0,0 +1,27 @@
+sudo: required
```

Comment:
I didn't realize that we had already stopped publishing this module to PyPI (Ref https://github.com/WikiWatershed/rapid-watershed-delineation/commit/21d784579c708134aa233d8fd0f294e10f24ecda). That makes things a lot simpler.